src/main/java/com/twentyonec/ItemsLogger/ItemsLogger.java

```diff
@@ -65,4 +67,22 @@ public void debugMessage(final String message) {
 		if (config.getDebug())
 			plugin.getLogger().log(Level.INFO, "[DEBUG]: " + message);
 	}
+
+	private void loadClasses() {
+		debugMessage("Loading classes");
+		if (config.getDeath()) {
+			debugMessage("Registering Death Event");
+			this.getServer().getPluginManager().registerEvents(new DeathSave(), this);
+		}
+		if (config.getJoin()) {
+			debugMessage("Registering Join Event");
+			this.getServer().getPluginManager().registerEvents(new JoinSave(), this);
+		}
+		if (config.getQuit()) {
+			debugMessage("Registering Quit Event");
+			this.getServer().getPluginManager().registerEvents(new QuitSave(), this);
+		}
+		this.getCommand("itemslogger").setExecutor(new ViewLogList());
+		this.getCommand("openitemlog").setExecutor(new OpenItemLog());
+	}
```

Comment:
Separate into two methods, one for setting up events and another for registering commands. Name them clearly, like _registerEvents_ and _registerCommands_.

Also, register all events on start and check whether it's enabled in the event itself. This way, you won't have to restart the whole server if somebody enables new logging mechanism on config reload. You aren't really doing any meaningful optimisation here anyway.